src/index.js

```diff
@@ -50,6 +52,15 @@ plotAndZoom(getHash());
 // Also when the event filters change
 onHashChange(plotAndZoom);
 
+// Now poll for new events data and when it arrives, update data 
+pollForNewEvents(30000, (err) => {
```

Comment:
Here we are reloading data every 30 seconds by re-fetching the script. We could have the interval be smaller but there is an issue: because we are loading the data as a script tag, when the server responds with 304/not modified, the client won't refetch it, *but* our callback has no way of knowing that the data is unmodified without comparing two enormous arrays by value, cause the browser won't tell us. 

We have to either do _that_, or we have to just update the app state regardless, which could trigger a bunch of other expensive crap.

By contrast, if we fetch this data as JSON via xhr, we *can* know when it's a 304 and just not do anything. We could refetch the data much more frequently without causing trouble.

Anyway, if 30 seconds is fine, I'd say probably whatever, this is simple enough.